lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -284,7 +284,7 @@ def define
 
       outcome :outcome_same_sex_civil_partnership_affirmation
 
-      outcome :outcome_os_marriage_impossible_no_laos_locals
+      outcome :outcome_opposite_sex_marriage_laos_with_no_lao_national
```

Comment:
Minor: I don't find this outcome name very readable. `outcome_opposite_sex_marriage_laos_with_non_lao_national` or `outcome_opposite_sex_marriage_laos_without_lao_national` might be better, but they might not mean the right thing.